src/main/scala/play/doc/FileRepository.scala

```diff
@@ -5,7 +5,7 @@ import java.io.File
 import java.io.InputStream
 import java.util.jar.JarFile
 import java.util.zip.ZipEntry
-import scala.collection.JavaConverters._
+import scala.collection.JavaConverters.*
```

Comment:
OK, I understand you wanted to remove all warnings that appear in Scala 3.
However it will not be possible to resolve all warnings, because e.g. `import ...*` will just not work in Scala 2.
So we will have to life with such warnings until we completely drop Scala 2 support.

Reply:
fwiw, both Scala 2.12.17 and Scala 2.13.10 accept the `*` under `-Xsource:3`

Reply:
Thanks, didn't know that.